Blueprints/BlueprintDefinitions/netcore3.1/template.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>Amazon.Lambda.Templates</id>
-    <version>4.1.2</version>
+    <version>4.1.3</version>
```

Comment:
Lets bump this to 4.2.0 since it isn't a bug fix and more of a feature.

Reply:
Done